example.env

```diff
@@ -15,4 +15,5 @@ USERSERVICES_PORT = 3031
 ORDERCATALOG_PORT = 3032
 ORDERSERVICE_PORT = 3033
 NOTIFICATIONSERVICE_PORT = 3034
-PAYMENTSERVICE_PORT = 3035
\ No newline at end of file
+PAYMENTSERVICE_PORT = 3035
+BOOKSERVICE = 3036
```

Comment:
Just add the "port" so we know this is a port reference variable (for someone new, if they found this env variable in other part of the code
"BOOKSERVICE_PORT"
It just might looks (rendundante) but its a clarification for future coders in the project.

Reply:
@ManasesLovera